kvbc/src/reconfiguration_kvbc_handler.cpp

```diff
@@ -355,39 +319,25 @@ bool ReconfigurationHandler::handle(const concord::messages::ClientKeyExchangeSt
         if (saved_ts.has_value()) {
           auto strval = std::visit([](auto&& arg) { return arg.data; }, *saved_ts);
           numeric_ts = concordUtils::fromBigEndianBuffer<uint64_t>(strval.data());
-          if (!command.tls) {
-            stats.timestamps.push_back(std::make_pair(cid, numeric_ts));
-          }
+          stats.timestamps.push_back(std::make_pair(cid, numeric_ts));
         }
         auto res =
             ro_storage_.get(concord::kvbc::categorization::kConcordReconfigurationCategoryId, key, bid.value().version);
         if (res.has_value()) {
           auto strval = std::visit([](auto&& arg) { return arg.data; }, *res);
-          if (!command.tls) {
-            concord::messages::ClientExchangePublicKey cmd;
-            std::vector<uint8_t> bytesval(strval.begin(), strval.end());
-            concord::messages::deserialize(bytesval, cmd);
-
-            LOG_INFO(getLogger(), "found transactions public key exchange status for client" << KVLOG(cid));
-            stats.clients_data.push_back(std::make_pair(cid, cmd.pub_key));
-          } else {
-            concord::messages::ClientTlsExchangeKey cmd;
-            std::vector<uint8_t> bytesval(strval.begin(), strval.end());
-            concord::messages::deserialize(bytesval, cmd);
-
-            LOG_INFO(getLogger(), "found tls certificate exchange status for client" << KVLOG(cid));
-            for (const auto& [icid, cert] : cmd.clients_certificates) {
-              stats.clients_data.push_back(std::make_pair(icid, cert));
-              stats.timestamps.push_back(std::make_pair(icid, numeric_ts));
-            }
-          }
+          concord::messages::ClientExchangePublicKey cmd;
+          std::vector<uint8_t> bytesval(strval.begin(), strval.end());
+          concord::messages::deserialize(bytesval, cmd);
+
+          LOG_INFO(getLogger(), "found transactions public key exchange status for client" << KVLOG(cid));
+          stats.clients_data.push_back(std::make_pair(cid, cmd.pub_key));
         }
       }
     }
   }
   rres.response = stats;
   return true;
-}
+}  // namespace concord::kvbc::reconfiguration
```

Comment:
Isn't this the end of a function? Not a namespace?

Reply:
fixed